bayesian/api/api_v2.py

```diff
@@ -211,6 +211,12 @@ def stack_analyses_with_request_id(external_request_id):
     """Handle stack analyses report fetch api."""
     start = time.time()
     logger.debug("[GET] /stack-analyses/%s", external_request_id)
+    # Revert after RDS upgrade.
+    if True == True:
```

Comment:
It will misleading, it should be empty instead of dummy data, even if user runs for other ecosystems he/she will get python data in response.

Reply:
yes.But @yzainee suggestion was to keep dumy data.
I am okay with naything.Either way UTs will fail.

Reply:
Yes, Yusuf suggested to have a dummy data, which will have empty for all fields, so clients will not fail because of missing keys but also show 0/empty for all analysis

Reply:
Ahh okay.

Reply:
Even this custom data should be ok. WDYT @jparsai ?

Reply:
fixed it